### PR TITLE
Update indexer.ts

### PR DESCRIPTION
**Pull request by bichikim**
- fix loadSvelteConfig reading error in some cases

### CODE REVIEW

**Comment by JReinhold** [comment]
Thanks! Can you share an example of when this would be necessary?

**Comment by bichikim** [comment]
Please check the link below :) 

https://github.com/bichikim/addon-svelte-csf-error-example

File: src/preset/indexer.ts [hunk 1]
```diff
@@ -9,7 +9,12 @@ import { loadSvelteConfig } from '../config-loader.js';
 
 export async function readStories(fileName: string) {
   let code = (await fs.readFile(fileName, 'utf-8')).toString();
-  const svelteOptions = await loadSvelteConfig();
+  const loadedSvelteOptions = await loadSvelteConfig()
+
+  /**
+   * loadedSvelteOptions can be {default: {...}, configFile: '...path'}
+   */
+  const svelteOptions = loadedSvelteOptions.default ? loadedSvelteOptions.default : loadedSvelteOptions;
```
Comment (JReinhold) [inline src/preset/indexer.ts:17]:
```suggestion
  const svelteOptions = loadedSvelteOptions?.default ? loadedSvelteOptions.default : loadedSvelteOptions;
```

**Comment by JReinhold** [comment]
Thanks for the reproduction! I don't fully understand when it has `default` and when it doesn't - most likely ESM/CJS related - but this looks like a rather defensive fix anyway.

**Comment by shilman** [comment]
:rocket: PR was released in [`v4.1.6`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v4.1.6) :rocket:

**Pull request merged**